scripts/gitlab/build_and_test.sh

```diff
@@ -12,5 +12,22 @@ if [ $? -ne 0 ]; then { echo "ERROR: build_src.py failed." ; exit 1; } fi
 if [[ "$DO_INTEGRATION_TESTS" == "yes" ]] ; then
     cd *build_and_test_*/build-*
     ./ats.sh
-    if [ $? -ne 0 ]; then { echo "ERROR: ATS failed." ; exit 1; } fi
+    if [ $? -ne 0 ]; then 
+        # Print each failing test's logs
+        cd *.*.logs
+        FAILING_TEST_NUMBERS=$(awk '/#[0-9]* FAIL/ {print $1}' ats.log | cut -c 2-)
+        for num in $FAILING_TEST_NUMBERS ; do
+            LOG_FILENAMES=(*$num*.log *$num*.log.err)
+            for f in ${LOG_FILENAMES[@]} ; do
+                if [ -s $f ] ; then
+                    echo "\n======== START $f START ========"
```

Comment:
```suggestion
                    echo ; echo "======== START $f START ========"
```
Apparently echo doesn't like '\n'.  This is what you get for trusting random things I send you in teams that I didn't bother to test. 

Apparently you can also use `printf "\n foo"` but `echo -e "\n foo"` isn't on all *nix platforms.